gradle.properties

```
@@ -1,5 +1,5 @@
-version=7.2.1
-iexecCommonVersion=6.1.0
+version=7.2.2-rc2
```

Do not forget before merging :)
```suggestion
version=7.2.2
```

---

done in a43eb67